winml/test/api/APITest.h

```diff
@@ -13,14 +13,14 @@ static void LoadModel(const std::wstring& modelPath,
   learningModel = winml::LearningModel::LoadFromFilePath(fullPath);
 };
 
-static uint64_t GetAdapterIdQuadPart(winml::LearningModelDevice& device) {
+static inline uint64_t GetAdapterIdQuadPart(winml::LearningModelDevice& device) {
```

Comment:
Just curious, what's the difference between "static inline" and "static" in this case?

Reply:
"static" makes it private linkage (only visible in this compilation unit/not able to reference from other source files during linkage), "inline" will suggest to the compiler that it should inline the method (insert the code at the call site instead of making a function call).

Though the specific reason why I added "inline" was to silence a MSVC warning - not all files that include APITest.h call GetAdapterIdQuadPart, and if you implement a non-inline function that has private linkage (i.e. only visible in the current file) and never call it, you get `C5245: unreferenced function with internal linkage has been removed`

Reply:
Sounds like we need to make a rule: all function definitions in *.h files must be declared with "inline"